arangod/Cluster/ClusterComm.h

```diff
@@ -697,15 +704,17 @@ class ClusterComm {
   /// @brief our background communications thread
   //////////////////////////////////////////////////////////////////////////////
 
-  ClusterCommThread* _backgroundThread;
+  std::atomic_uint _roundRobin;
+  std::vector<ClusterCommThread*> _backgroundThreads;
+
+  std::shared_ptr<communicator::Communicator> communicator();
```

Comment:
Forward declaration not needed here, as the implementation follows in lines 740ff.